x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -312,6 +330,11 @@ export interface ActionListApiResponse {
   endDate: string | undefined;
   userIds: string[] | undefined; // users that requested the actions
   commands: string[] | undefined; // type of actions
-  data: ActionDetails[];
+  /**
+   * The `outputs` is not currently part of the list response due to possibly large amounts of
+   * data, especially for cases (in the future) where we might support actions being sent to
+   * multiple agents
+   */
+  data: Array<Omit<ActionDetails, 'outputs'>>;
```

Comment:
this is an adjustment to the type for the return data of the Action List API. We don't return `outputs` in the list API (by design)